src/controllers/game.js

```diff
@@ -1,3 +1,3 @@
 exports.get = (req, res) => {
-  res.render('game');
+  res.render('game', { g: true });
```

Comment:
we should use readable with meaning variable names ,, not like g, h or any other letter